### ISSUE TITLE
Support multiple subscript operator in proxified PyObjects.

### ISSUE THREAD

**Issue by antl3x** [mmomtchev/pymport#101]
The following way to use multiple values below work:

```javascript
df.item(['Age', 'Fare']).median().type
'Series'
```

Is possible to support something without relying in manual calling the .item() ?

This **works**:

```javascript
df['Age'].median().type
'numpy.float64'
``` 

This **is wrong** (since should be a Series, and not a float value):

```javascript
df['Age', 'Fare'].median().type
'numpy.float64'
``` 

This **does not work:**
```javascript
df[['Age', 'Fare']].median().type
Uncaught TypeError: Cannot read properties of undefined (reading 'median')
```

**Comment by mmomtchev** [comment]
Not really. `item()` exists for a reason. In JavaScript the subscript operator is much more limited. It accepts only scalars.

For example this:
```js
p['a', 'b']
```
is functionally equivalent to
```js
p['b']
```

`'a', 'b'` evaluated in a scalar context is `'b'`.

There are cases where using `[]` works, but it is not supported and you are on your own risk. Only a function as `.item()` can accept all the values that Python can accept.

**Comment by mmomtchev** [comment]
In case you haven't seen my profile, I am currently living on social welfare because an ex-employer is extorting me with the help of the French police and the French judiciary over a series of false judicial proceedings, including some very serious allegations, in order to cover up his intimate personal problem. In order to further intimidate me, people are simultaneously posting on my projects. Yours is one, it is posted simultaneously with a StackOverflow issue with similar content. You don't happen to know anything about this affair?
